app/assets/javascripts/room.js

```diff
@@ -46,6 +46,36 @@ $(document).on('turbolinks:load', function(){
       showUpdateRoom(this)
     })
 
+    // share room pop up accessibility
```

Comment:
Just one comment - can we create a function at the bottom with this code called `manageAccessAccessibility` and put all of this code there? Then we could replace all this code with a simple function call

Reply:
Thank you for reviewing, I have updated the code, please check it again if you have time.